rabbit-escape-engine/src/rabbitescape/engine/textworld/LineProcessor.java

```diff
@@ -399,6 +445,7 @@ else if ( value.equals( "false" ) )
 
     private void processItemsLine( String line, VariantGenerator variantGen )
     {
+        maybeLinkToLastComment( Comment.WORLD_ASCII_ART );
         // Treat empty lines as blank lines (Github converts blank lines to empty lines, so it seems sensible to reverse the process).
         if ( line.length() != 0 )
```

Comment:
Interspersing comments within the ASCII art block would be bad. It is good that this would correct that.